Vostok.Applications.AspNetCore/Builders/VostokKestrelBuilder.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets;
+using Vostok.Applications.AspNetCore.Configuration;
+using Vostok.Commons.Helpers;
+
+namespace Vostok.Applications.AspNetCore.Builders
+{
+    internal class VostokKestrelBuilder
+    {
+        private readonly Customization<KestrelSettings> kestrelCustomization = new Customization<KestrelSettings>();
+
+        public void Customize(Action<KestrelSettings> customization)
+            => kestrelCustomization.AddCustomization(customization);
+
+        public void ConfigureKestrel(KestrelServerOptions options)
+        {
+            var settings = kestrelCustomization.Customize(new KestrelSettings());
+
+            options.AddServerHeader = false;
+            options.AllowSynchronousIO = false;
+
+            options.Limits.MaxConcurrentConnections = null;
+            options.Limits.MaxRequestBufferSize = 256 * 1024;
+            options.Limits.MaxResponseBufferSize = 256 * 1024;
+
+            #if NETCOREAPP3_1
+            options.Limits.Http2.MaxStreamsPerConnection = 1000;
```

Comment:
Q: Is there any reason to override defaults? Performance considerations, etc. ? Related to buffers, max streams.